sema4ai-http-helper/pyproject.toml

```diff
@@ -14,6 +14,8 @@ packages = [{ include = "*", from = "src" }]
 python = "^3.10"
 truststore = "0.9.2"
 urllib3 = ">=2.2.2"
+pyyaml = "6.0.2"
```

Comment:
Given that this is a helper to be included in other projects, make this more flexible (`>=5.0` would work).